### PR TITLE
rule updates

### PR DESCRIPTION
**Pull request by mr-tz**

### CODE REVIEW

File: data-manipulation/encryption/hc-128/encrypt-data-using-hc-128.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 rule:
   meta:
-    name: encrypt-data-using-hc-128
+    name: encrypt data using hc 128
```
Comment (williballenthin) [inline data-manipulation/encryption/hc-128/encrypt-data-using-hc-128.yml:3]:
```suggestion
    name: encrypt data using HC-128
```

File: data-manipulation/encryption/camellia/encrypt-data-using-camellia.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 rule:
   meta:
-    name: encrypt-data-using-camellia
+    name: encrypt data using camellia
```
Comment (williballenthin) [inline data-manipulation/encryption/camellia/encrypt-data-using-camellia.yml:3]:
```suggestion
    name: encrypt data using Camellia
```

File: data-manipulation/encryption/sosemanuk/encrypt-data-using-sosemanuk.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 rule:
   meta:
-    name: encrypt-data-using-sosemanuk
+    name: encrypt data using sosemanuk
```
Comment (williballenthin) [inline data-manipulation/encryption/sosemanuk/encrypt-data-using-sosemanuk.yml:3]:
```suggestion
    name: encrypt data using Sosemanuk
```

**Review by williballenthin — COMMENTED** [review]
some ideas

**Comment by mr-tz** [comment]
👍 thanks, awesome to commit these changes directly back from the files tab!

**Pull request merged**